api/getambassador.io/v2/Host.proto

```diff
@@ -96,3 +113,24 @@ message ACMEProviderSpec {
   // This is normally set automatically
   string registration = 4;
 }
+
+message RequestPolicy {
+  // How shall we handle insecure requests?
+  InsecureRequestPolicy insecure = 1;
+
+  // Later we may define a 'secure' section too.
+}
+
+message InsecureRequestPolicy {
+  // What action should be taken for an insecure request?
+  InsecureRequestAction action = 1;
+
+  // Is there an additional insecure port we should listen on?
+  uint32 additionalPort = 2;
```

Comment:
I'm assuming that the `additionalPort` will be rarely used, and that when it is used, it'll be by Julian -- that's why it's associated with a `Host`, and why it's explicitly "this is an insecure port" rather than "this is a cleartext port".

We do need it, to support the scenario of a split L4 load balancer -- this was spelled out in the doc about the `requestPolicy`, remember? 🙂